micro-infra-spring-base/src/test/groovy/com/ofg/infrastructure/web/resttemplate/MetricsAspectIntegrationSpec.groovy

```diff
@@ -0,0 +1,154 @@
+            verifyMetric { restTemplate.execute(wireMockUri, httpMethod, requestCallback, requestExtractor) }
+
+            verifyMetric { restTemplate.getForEntity(wireMockUrl, responseType) }
+            verifyMetric { restTemplate.getForEntity(wireMockUrl, responseType, [:]) }
+            verifyMetric { restTemplate.getForEntity(wireMockUri, responseType) }
+
+            verifyMetric { restTemplate.getForObject(wireMockUrl, responseType) }
+            verifyMetric { restTemplate.getForObject(wireMockUrl, responseType, [:]) }
+            verifyMetric { restTemplate.getForObject(wireMockUri, responseType) }
+
+            verifyMetric { restTemplate.headForHeaders(wireMockUrl) }
+            verifyMetric { restTemplate.headForHeaders(wireMockUrl, [:]) }
+            verifyMetric { restTemplate.headForHeaders(wireMockUri) }
+
+            verifyMetric { restTemplate.optionsForAllow(wireMockUrl) }
+            verifyMetric { restTemplate.optionsForAllow(wireMockUrl, [:]) }
+            verifyMetric { restTemplate.optionsForAllow(wireMockUri) }
+
+            verifyMetric { restTemplate.postForEntity(wireMockUrl, request, responseType) }
+            verifyMetric { restTemplate.postForEntity(wireMockUrl, request, responseType, [:]) }
+            verifyMetric { restTemplate.postForEntity(wireMockUri, request, responseType) }
+
+            verifyMetric { restTemplate.postForObject(wireMockUrl, request, responseType) }
+            verifyMetric { restTemplate.postForObject(wireMockUrl, request, responseType, [:]) }
+            verifyMetric { restTemplate.postForObject(wireMockUri, request, responseType) }
+
+            verifyMetric { restTemplate.postForLocation(wireMockUrl, request) }
+            verifyMetric { restTemplate.postForLocation(wireMockUrl, request, [:]) }
+            verifyMetric { restTemplate.postForLocation(wireMockUri, request) }
+
+            verifyMetric { restTemplate.put(wireMockUrl, request) }
+            verifyMetric { restTemplate.put(wireMockUrl, request, [:]) }
+            verifyMetric { restTemplate.put(wireMockUri, request) }
+    }
+
+    void verifyMetric(String expectedMetricNameSuffix = '', Closure closure) {
+        closure()
+        def expectedMetricNamePrefix = "RestOperations.localhost.$wireMockPort"
+        def expectedMetricName = expectedMetricNamePrefix + expectedMetricNameSuffix
+        assert metricRegistry.lastTimerMetricName() == expectedMetricName
+        metricRegistry.resetLastTimerMetricName()
+    }
+
+    @EnableAspectJAutoProxy
+    static class Config {
+
+        @Bean
+        static PropertySourcesPlaceholderConfigurer placeholderConfigurer() {
+            return new PropertySourcesPlaceholderConfigurer()
+        }
+
+        @Bean
+        MetricRegistry metricRegistry() {
+            def lastMetricName = 'UNSET'
+            return [
+                    timer              : { String metricName ->
```

Comment:
you mean the spaces after the timer or the missalignment with the last colon (line 145)?